runtime/src/main/java/org/corfudb/protocols/logprotocol/MultiObjectSMREntry.java

```diff
@@ -27,51 +26,42 @@
 public class MultiObjectSMREntry extends LogEntry implements ISMRConsumable {
 
     // map from stream-ID to a list of updates encapsulated as MultiSMREntry
-    @Getter
-    public Map<UUID, MultiSMREntry> entryMap = Collections.synchronizedMap(new HashMap<>());
+    //TODO(Maithem): we should hide entryMap
+    private final Map<UUID, List<SMREntry>> entryMap = Collections.synchronizedMap(new HashMap<>());
```

Comment:
I think we should also remove MultiSMREntry since its redundant with this container, but that can be done later.